addons/QuickSceneRunner/SceneRunner.gd

```diff
@@ -69,10 +70,10 @@ func edit_scene(scene: Control):
 
 func save_scenes():
 	var scene_list := PackedStringArray()
-	
```

Comment:
Unnecessary whitespace changes. I assume you have a setting enabled, but this shouldn't be commited.